fourth-wall.css

```diff
@@ -81,6 +81,9 @@ body {
 #pulls li.thumbsup {
     background: #5c9317;
 }
+#pulls li.changes-requested {
+    background: #BF1E2D !important;
```

Comment:
You shouldn’t need the `!important` here if this declaration is coming after any others that set the background on this element.